npm create pattern-lab with specified directory and starterkit broken

install starterkit needs to be `projectDir` aware

workaround:

`mkdir my-project && cd my-project` before you `npm create pattern-lab`
then, specify the current directory as the install target

Same for me as well. macOS 10.15.6, node 12.18, and in Ubuntu Docker image with node 10.19. 

The workaround mentioned above does not work for me. 

```
⊙ patternlab → Installing edition: @pattern-lab/edition-node(node:7701) UnhandledPromiseRejectionWarning: Error: Cannot find module '@pattern-lab/edition-node/package.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.resolve (internal/modules/cjs/helpers.js:33:19)
    at resolveFileInPackage (/root/.npm/_npx/7667/lib/node_modules/create-pattern-lab/node_modules/@pattern-lab/core/src/lib/resolver.js:10:18)
    at resolvePackageFolder (/root/.npm/_npx/7667/lib/node_modules/create-pattern-lab/node_modules/@pattern-lab/core/src/lib/resolver.js:18:23)
    at resolveDirInPackage (/root/.npm/_npx/7667/lib/node_modules/create-pattern-lab/node_modules/@pattern-lab/core/src/lib/resolver.js:26:20)
    at /root/.npm/_npx/7667/lib/node_modules/create-pattern-lab/node_modules/@pattern-lab/cli/bin/install-edition.js:38:7
    at Generator.next (<anonymous>)
    at spwn (/root/.npm/_npx/7667/lib/node_modules/create-pattern-lab/node_modules/@pattern-lab/cli/bin/utils.js:75:25)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:7701) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:7701) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

--- a/packages/cli/bin/cli-actions/init.js
+++ b/packages/cli/bin/cli-actions/init.js
@@ -46,6 +46,7 @@ const init = options =>
     ); // 1
 
     yield scaffold(projectDir, sourceDir, publicDir, exportDir); // 2
+    process.env.projectDir = path.join(process.cwd(), projectDir);
 
     if (edition) {
       spinner.text = `⊙ patternlab → Installing edition: ${edition}`;

--- a/packages/cli/bin/install-starterkit.js
+++ b/packages/cli/bin/install-starterkit.js
@@ -17,7 +17,10 @@ const installStarterkit = (starterkit, config) =>
     const sourceDir = config.paths.source.root;
     const name = starterkit.value || starterkit;
     yield checkAndInstallPackage(name);
-    yield copyAsync(resolveDirInPackage(name, 'dist'), path.resolve(sourceDir));
+    yield copyAsync(
+      resolveDirInPackage(name, 'dist'),
+      path.resolve(process.env.projectDir || '', sourceDir)
+    );
     let kitConfig;
     const kitConfigPath = path.join(
       resolvePackageFolder(name),

--- a/packages/cli/bin/utils.js
+++ b/packages/cli/bin/utils.js
@@ -135,7 +135,11 @@ const fetchPackage = packageName =>
     const installCmd = useYarn ? 'add' : 'install';
     try {
       if (packageName) {
-        const cmd = yield spawn(pm, [installCmd, packageName]);
+        const opts = {};
+        if (process.env.projectDir) {
+          opts.cwd = process.env.projectDir;
+        }
+        const cmd = yield spawn(pm, [installCmd, packageName], opts);
         error(cmd.stderr);
       }
     } catch (err) {

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -16,7 +16,7 @@
     "chalk": "2.4.1",
     "commander": "2.15.1",
     "deepmerge": "^2.1.1",
-    "execa": "0.10.0",
+    "execa": "4.0.3",
     "fs-extra": "6.0.1",
     "glob": "7.1.2",
     "has-yarn": "1.0.0",

